### PR TITLE
Cut 2.2.0-rc.0

### PR DESCRIPTION
**Pull request by brancz**
@fabxc @Gouthamve @brian-brazil @krasi-georgiev

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,20 @@
-## 2.1.0 / 2018-1-19
+## 2.2.0-rc.0 / 2018-02-12
+
+* [CHANGE] Set consul server default to `localhost:8500`.
+* [CHANGE] Rename file SD mtime metric.
+* [CHANGE] Change fule file formatting.
```
Comment (krasi-georgiev) [inline CHANGELOG.md]: what is `fule file formatting.` ?

Comment (brancz) [reply CHANGELOG.md]: rule file formatting :smile:

**Comment by krasi-georgiev** [comment]
probbaly not important but can also add this one
https://github.com/prometheus/prometheus/pull/3821

up ==bool 1

before
up{instance="localhost:9090",job="prometheus"}
after
{instance="localhost:9090",job="prometheus"}

File: CHANGELOG.md (hunk 1 above)
Comment (krasi-georgiev) [reply CHANGELOG.md]: I even did few google/github searches, thinking I am missing something important :smile_cat:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,20 @@
-## 2.1.0 / 2018-1-19
+## 2.2.0-rc.0 / 2018-02-12
+
+* [CHANGE] Set consul server default to `localhost:8500`.
```
Comment (brian-brazil) [inline CHANGELOG.md]: Enhancement

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+* [CHANGE] Rename file SD mtime metric.
+* [CHANGE] Change rule file formatting.
```
Comment (brian-brazil) [inline CHANGELOG.md]: This is a UI change, it's an enhancement

**Comment by brancz** [comment]
@krasi-georgiev added it as a `[CHANGE]` let me know if you think that applies.

**Comment by krasi-georgiev** [comment]
yep , I can't think of anything else so looks good I think.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,21 @@
-## 2.1.0 / 2018-1-19
+## 2.2.0-rc.0 / 2018-02-12
+
+* [CHANGE] Rename file SD mtime metric.
+* [CHANGE] Drop metric name for bool modifier.
```
Comment (brian-brazil) [inline CHANGELOG.md]: This is a bugfix

**Comment by brancz** [comment]
I think I applied all requested changes and changed the date to today. Let me know if there is anything else.

**Comment by brian-brazil** [comment]
:+1:

**Comment by fabxc** [comment]
👍

**Pull request merged**